### PR TITLE
Anya/2693 update brieff

### PR DESCRIPTION
**Pull request by aroltsch**
connects #2693

### CODE REVIEW

File: app/mappers/hearing_mapper.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    case hearing_record.hearing_disp
+    when "H"
+      return "1" if case_record.bfhr == "1"
+      return "2" if case_record.bfhr == "2" && case_record.bfdocind != "V"
+      return "6" if case_record.bfhr == "2" && case_record.bfdocind == "V"
+    when "P"
+      nil
+    when "C"
+      "5"
+    else
```
Comment (cmgiven) [inline app/mappers/hearing_mapper.rb]: Can we be explicit that this is for `"N"`? Don't want that to get lost.

File: spec/mappers/hearing_mapper_spec.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      it { is_expected.to eq "1" }
+    end
+
+    context "when disposition is held and it is video hearing" do
+      let(:hearing_disp) { "H" }
+      let(:bfhr) { "2" }
+      let(:bfdocind) { "V" }
+
+      it { is_expected.to eq "6" }
+    end
```
Comment (cmgiven) [inline spec/mappers/hearing_mapper_spec.rb]: Can we add the other cases as well?

```
"when the disposition is held and it is a travel board hearing"
"when the disposition is postponed"
"when the disposition is canceled"
"when the disposition is no-show"
```

File: app/mappers/hearing_mapper.rb (hunk 1 above)
Comment (aroltsch) [inline app/mappers/hearing_mapper.rb]: Good idea!

File: spec/mappers/hearing_mapper_spec.rb (hunk 2 above)
Comment (aroltsch) [inline spec/mappers/hearing_mapper_spec.rb]: I can do that - yes!

File: spec/mappers/hearing_mapper_spec.rb [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+      let(:bfhr) { "2" }
+      let(:bfdocind) { nil }
+
+      it { is_expected.to eq "2" }
+    end
+
+    context "when disposition is postponed" do
+      let(:hearing_disp) { "P" }
+      let(:bfhr) { "1" }
+      let(:bfdocind) { "V" }
```
Comment (cmgiven) [inline spec/mappers/hearing_mapper_spec.rb]: This combination of 1 and V should never happen. But that's nitpicking.

**Pull request merged**